proto/lbm/fswap/v1/event.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package lbm.fswap.v1;
+
+option go_package = "github.com/Finschia/finschia-sdk/x/fswap/types";
```

Comment:
It's not mandatory, but why don't we refer to foundation's pacakge structure.
```suggestion
option go_package = "github.com/Finschia/finschia-sdk/x/fswap";
```

Reply:
I found that zkauth/da in finschia-sdk and the most of packages in latest cosmos-sdk are generate proto files in types. So I think it is better than previous design.
https://github.com/Finschia/finschia-sdk/tree/feat/zkauth/x/zkauth/types
https://github.com/Finschia/finschia-sdk/tree/for-ramus/x/or/da/types
https://github.com/cosmos/cosmos-sdk/tree/main/x

Reply:
We temporarily store the generated proto files under the type directory, and then make corresponding adjustments based on the final development of relayer/bridge.